src/api/routes/foodphotos.ts

```diff
@@ -0,0 +1,58 @@
+import express, { Application, Request, Response, NextFunction } from "express";
+const data = require("../../../data/photofooddata.json");
+
+const router = express.Router();
+
+/*
+
+/photos/food/taste -> send all the taste category
+
+
+/photos/food -> send all the food items
+
+
+/photos/food?taste={category} -> send items of specific category food
+
+
+*/
+
+// GET request /photos/food/taste route -> available category
+router.get("/taste", (req: Request, res: Response, next: NextFunction) => {
+  try {
+    res.status(200).json(data.taste);
+  } catch (err) {
+    //error handling
+    console.log(err);
+    res.status(500).json({
+      error: err,
+    });
+  }
+});
+
+// GET request /photos/food route
+router.get("/", (req: Request, res: Response, next: NextFunction) => {
+  try {
+    //if query parameters does not exist then send all food items
+    if (!req.query.taste) {
+      res.status(200).json(data.data);
```

Comment:
The endpoint should be consistent with the format of the `/photos` endpoint. Have a look at [this](https://github.com/adityabisoi/REST-fetch/pull/70).

The response format is:
```js
{
	"data": [
		... // The objects
	]
}
```

What your code would return is:
```js
[
	... // The objects
]
```

So what you need to do is:
```js
res.status(200).json({ data: data.data });
```

This update is needed to ensure consistency with the `/photos` endpoint, since `/photos/food` is an extension to the same.